src/lib/redux/customerData.ts

```diff
@@ -143,3 +150,15 @@ export const selectCustomersToCollect = createSelector(
         return customersToCollect;
     }
 )
+
+export const selectCustomersDone = createSelector(
```

Comment:
I'm not super familiar with the various use cases, but I'm wondering if you could break this selector down into smaller pieces

Reply:
I think this is fine for now. We can put a comment as a future TODO, but I can't think of anything cleaner off the top of my head